R/iso_load.R

```diff
@@ -0,0 +1,56 @@
+# read cached isoreader files
+# @param ds the data structure to fill
+iso_load <- function(ds) {
```

Comment:
these functions are not usually called, they are all dispatched from `iso_read_continuous_flow` and `iso_read_dual_inlet` so the name does not really matter but for consistency I suggest keeping this as `iso_read_rds`. For details on the reader registration, I started adding a section to the [advanced operations vignette](http://isoreader.kopflab.org/articles/advanced.html) that you may find informative. It's not complete yet so not linked on the official isoreader docu but getting there slowly.